install-odoo-saas.sh

```diff
@@ -26,7 +26,7 @@
  export ODOO_DATA_DIR=${ODOO_DATA_DIR:-"/opt/odoo/data/"}
  export BACKUPS_DIR=${BACKUPS_DIR:-"/opt/odoo/backups/"}
  export LOGS_DIR=${LOGS_DIR:-"/var/log/odoo/"}
- export OPENERP_SERVER=${OPENERP_SERVER:-/etc/openerp-server.conf}
+ export ODOO_SERVER=${ODOO_SERVER:-/etc/odoo-server.conf}
```

Comment:
I'd prefer to keep OPENERP_SERVER or use ODOO_RC. Because only these two are supported by odoo

Reply:
The purpose was mainly to change the configuration file name to something correct and convenient, it is a detail but since opens ERP is now non-existent !

Reply:
It makes sense. But use ODOO_RC instead of ODOO_SERVER